user_manual/android_app.rst

```diff
@@ -239,6 +251,42 @@ Just ask them to copy and paste and send it to you.
 .. figure:: images/android-14.png
    :alt: Federated share creation.
    
+Working With Multiple Files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Android application can perform some operations on multiple files simultaneously, such as refreshing and deleting. 
+To select multiple files, long select the first file that you want to work with; you will see a checkbox appear on the far right-hand side. 
+After that, check the checkbox next to all the other files that you want to perform the same operation on, and then perform the operation.
+
+.. figure:: images/android-select-multiple-files.png
+   :alt: Select multiple files.
+   
+Uploading Files Taken From the Camera
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Pictures and videos can be uploaded from your smartphone after choosing the
+folder where they are stored.
+To specify where they are located, in the "Settings" options, first enable one of "Instant picture uploads" or "Instant video uploads". 
+After that, a further option called "Camera folder" will become visible, as in the screenshot below. 
+
+.. figure:: images/specify-camera-folder.png
+   :alt: Specify camera folder.
+   
+Make Folders Available Offline 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Folders can be made available when no internet or mobile connectivity is available.
+Doing so caches a copy of the folder and its contents locally to the device, assuming that sufficient disc space is available.
+Making folders available offline works slightly differently, depending on the number of folders selected for offline availability. 
```

Comment:
Not only with a lack of connectivity, "available offline" assures that the file content will be always synced. This is important to clear, so a downloaded file/folder (not an available offline one), is also available when the device does not have network connectivity.